{"name": "test_firstTest", "status": "failed", "statusDetails": {"message": "AssertionError: assert False", "trace": "self = <test_myFirstTest.TestMush object at 0x1053633e0>\n\n    def test_firstTest(self):\n        print('First test1')\n        print('Second test')\n>       assert False\nE       assert False\n\nsrc/test_myFirstTest.py:5: AssertionError"}, "attachments": [{"name": "stdout", "source": "f0142dcc-901f-4d6c-8244-549b32940624-attachment.txt", "type": "text/plain"}], "start": 1745046059113, "stop": 1745046059113, "uuid": "8f48dd70-7a1d-4c60-b483-4cf21e5b145b", "historyId": "98b9d2a354c1b40711ae7fd21ed656bb", "testCaseId": "98b9d2a354c1b40711ae7fd21ed656bb", "fullName": "src.test_myFirstTest.TestMush#test_firstTest", "labels": [{"name": "parentSuite", "value": "src"}, {"name": "suite", "value": "test_myFirstTest"}, {"name": "subSuite", "value": "TestMush"}, {"name": "host", "value": "Musheer-Khan.local"}, {"name": "thread", "value": "54316-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "src.test_myFirstTest"}]}